repository/schema-registry/operator/templates/deployment.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Name }}
+  namespace: {{ .Namespace }}
+  labels:
+    app: schema-registry
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: schema-registry
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: schema-registry
+    spec:
+      containers:
+      - name: schema-registry
+        image: mesosphere/cp-schema-registry:5.3.0
```

Comment:
and use the confluent image

Reply:
Done. Changed the image to Confluentic one.

Reply:
and let's use the version `5.3.2`

Reply:
Done.